homeassistant/components/mullvad/config_flow.py

```diff
@@ -0,0 +1,23 @@
+"""Config flow for Mullvad VPN integration."""
+import logging
+
+from homeassistant import config_entries
+
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Mullvad VPN."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+
+        if user_input is not None:
+            return self.async_create_entry(title="Mullvad VPN", data=user_input)
+
+        return self.async_show_form(step_id="user")
```

Comment:
I think this misses logic to prevent the user from creating multiple config entries?

Reply:
Lets shoot for Gold!
![image](https://user-images.githubusercontent.com/10717998/89245647-46290600-d5d7-11ea-9f73-dc652717b772.png)